hudi-utilities/pom.xml

```diff
@@ -215,6 +216,14 @@
           <groupId>javax.servlet</groupId>
           <artifactId>*</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-client-api</artifactId>
+        </exclusion>
```

Comment:
Any dependency that we are excluding in sub modules, please review whether we can simply exclude it in the root POM itself. It will avoid these random code changes throughout the project.

Reply:
ack